src/app-directory/specification/appd.yaml

```diff
@@ -936,5 +985,11 @@ components:
               },
               Web App Manifest: https://example.com/fdc3-workbench.json
             }
+            localizedVersions: {
+              fr-FR: {
+                title: FDC3 Table de travail,
+                description: Outil de développement et de test pour les desktop agents et applications FDC3
+              }
+            }
```

Comment:
This is supposed to be here right? Just wondering cause it seems like it might have belonged in https://github.com/finos/FDC3/pull/670

Reply:
yep - these PRs keep changing the structure of the record so they are going into a consolidated branch (where the plan for the change was clear). We'll agree the less clear ones individually, merge them into the consolidated branch and then present the whole thing to the standards working group